ladderly-io/prisma/schema.prisma

```diff
@@ -39,6 +39,8 @@ model User {
   journalReminderEnabled      Boolean           @default(false)
   journalReminderFrequency    ReminderFrequency @default(WEEKLY)
   journalReminderLastReminded DateTime?
+  journalReminderHour         Int?
```

Comment:
I wonder why we need ReminderHour? isn't the below good enough

journalReminderLastReminded

and

enum ReminderFrequency {
  NONE
  DAILY
  WEEKLY
  MONTHLY
}

Reply:
hm. no i think it is still not needed. going to look in the cron logic. i think we can still catch based on journalReminderLastReminded

Reply:
oh. i see again. it isn't needed for cron logic, it is just a feature user is likely to desire. they can pick the reminder time of day

Reply:
this is smart but i want to support vercel free / hobby tier which only runs 2 times per day, so i won't let users customize it atm

https://vercel.com/docs/cron-jobs/usage-and-pricing